astro-docs/src/content/docs/reference/Deprecated/affected-graph.mdoc

```diff
@@ -3,6 +3,8 @@ title: 'affected:graph - CLI command'
 description: 'Graph dependencies affected by changes'
 sidebar:
   hidden: true
+weight: .1
+filter: "type:References"
```

Comment:
basically made the deprecated items never show up unless you hit the exact title phrase as they were over taking the results. which I think is okay as no reason to really return deprecated commands often.